### DISCUSSION TITLE
Sadle seat, mainly for hovercrafts (and mechs?)

### DISCUSSION THREAD

**Discussion by RedigTuffing** [tsunamayo/Starship-EVO#4646] [Suggestions]
Simply a seat option that allows the player character to have their legs on either side of the seat.
Maybe one with the legs streight (biker stance) and one where the legs are bent (sports bike stance) and a slight forward lean?

Just a suggestion.

**Comment by ZachZent** [comment]
Also a standing seat like the old ones we used to have. For mechs, escape pods, medical tables, beds, etc.
